src/_core/Settings.js

```diff
@@ -14,18 +14,19 @@ class Settings {
     });
   }
 
-  create (section, key, defaultOpt, name, description, id, cb) {
+  create(section, key, defaultOpt, name, description, id, author, cb) {
     if (!(section in queue)) {
       queue[section] = {};
       queue[section].items = [];
       queue[section].running = false;
     }
 
-    queue[section].items.push({ "key": key, "defaultOpt": defaultOpt, "name": name, "description": description, "id": id, "cb": cb, "purpose": "create" });
+    queue[section].items.push({ "key": key, "defaultOpt": defaultOpt, "name": name, "description": description, "id": id, "author": author, "cb": cb, "purpose": "create" });
     this.processQueue(section);
   }
 
-  update (section, key, setting, value) {
+  // TODO: Update all attributes so if an extension update moves a feature, it will update
```

Comment:
I think this probably makes more sense on `getFeatureSettings`, detect if something has been added/removed/modified and then update accordingly. Add/remove is straightforward, not sure about something like renaming.

Reply:
Moved it.